### DISCUSSION TITLE
posts/1_%E8%AF%91%E5%A6%82%E4%BD%95%E5%9C%A8web%E5%BC%80%E5%8F%91%E4%B8%AD%E4%BD%BF%E7%94%A8python/

### DISCUSSION THREAD

**Discussion by giscus** [rainfd/rainfd.github.io#2] [Announcements]
# posts/1_%E8%AF%91%E5%A6%82%E4%BD%95%E5%9C%A8web%E5%BC%80%E5%8F%91%E4%B8%AD%E4%BD%BF%E7%94%A8python/

原文：HOWTO Use Python in the Web
这篇文档展示了Python如何融入到web中。它介绍了几种Python结合web服务器的方法，以及开发网站的一些常规做法。

http://localhost:8000/posts/1_%E8%AF%91%E5%A6%82%E4%BD%95%E5%9C%A8web%E5%BC%80%E5%8F%91%E4%B8%AD%E4%BD%BF%E7%94%A8python/

**Comment by rainfd** [comment]
giscus comment test
```python
print("Hello, World!")
```
